The option 'send-analytics' contains an illegal character : '-'

https://github.com/googleapis/java-spanner/pull/452/checks?check_run_id=1456675000 ("view raw log" from the dropdown at the top-right)

```
2020-11-26T01:46:55.5767983Z + java -jar linkage-monitor-latest-all-deps.jar com.google.cloud:libraries-bom
2020-11-26T01:46:55.6234944Z ##[error]Exception in thread "main" java.lang.IllegalArgumentException: The option 'send-analytics' contains an illegal character : '-'
2020-11-26T01:46:55.6247558Z 	at org.apache.commons.cli.OptionValidator.validateOption(OptionValidator.java:71)
2020-11-26T01:46:55.6248848Z 	at org.apache.commons.cli.Option.<init>(Option.java:146)
2020-11-26T01:46:55.6249816Z 	at org.apache.commons.cli.Options.addOption(Options.java:150)
2020-11-26T01:46:55.6250893Z 	at org.apache.commons.cli.Options.addOption(Options.java:131)
2020-11-26T01:46:55.6253365Z 	at com.google.cloud.tools.dependencies.linkagemonitor.LinkageMonitor.parseCommandLine(LinkageMonitor.java:124)
2020-11-26T01:46:55.6256397Z 	at com.google.cloud.tools.dependencies.linkagemonitor.LinkageMonitor.main(LinkageMonitor.java:96)
```

--- a/linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java
+++ b/linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java
@@ -119,9 +119,10 @@ public class LinkageMonitor {
     }
   }
 
-  private static CommandLine parseCommandLine(String[] arguments) {
+  @VisibleForTesting
+  static CommandLine parseCommandLine(String[] arguments) {
     Options options = new Options();
-    options.addOption("send-analytics", false, "track results from each run");
+    options.addOption("sa", "send-analytics", false, "track results from each run");
     CommandLineParser parser = new DefaultParser();
     try {
       CommandLine commandLine = parser.parse(options, arguments);

--- a/linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java
+++ b/linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java
@@ -104,6 +104,12 @@ public class LinkageMonitorTest {
                 "(Lcom/google/protobuf/Message;)Lio/grpc/MethodDescriptor$Marshaller;",
                 false));
   }
+  
+  @Test
+  public void testCommandLine() {
+	String[] arguments = {"something"};
+	LinkageMonitor.parseCommandLine(arguments);
+  }
 
   @Test
   public void testBomSnapshot()

--- a/linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java
+++ b/linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java
@@ -107,8 +107,8 @@ public class LinkageMonitorTest {
   
   @Test
   public void testCommandLine() {
-	String[] arguments = {"something"};
-	LinkageMonitor.parseCommandLine(arguments);
+    String[] arguments = {"something"};
+    LinkageMonitor.parseCommandLine(arguments);
   }
 
   @Test

